Dockerfile

```diff
@@ -1,21 +1,24 @@
 # To enable ssh & remote debugging on app service change the base image to the one below
 # FROM mcr.microsoft.com/azure-functions/node:3.0-appservice
-FROM mcr.microsoft.com/azure-functions/node:3.0
+# FROM mcr.microsoft.com/azure-functions/node:3.0
+FROM mcr.microsoft.com/azure-functions/node:3.0-node12
```

Comment:
QQ: Why are we specifying a node12 specific version of the container here?

Reply:
3.0 defaults to node 10 on the machine. Thought that keeping our repos on 12 or 14 would be better and reduces the complexities of managing different node versions.

Reply:
Yep makes sense, thanks Leo!